GraphDistance/GraphDistance/Graph/GraphFile.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GraphDistance
+{
+    public static class GraphFile
+    {
+        public static Graph Read(string filePath)
```

Comment:
I don't remember if it should be .txt file or copy paste method. Btw does it work only with full directory?

Reply:
To się najwyżej doda inne metody, np. do wpisywania z palca.
Ścieżka względna, ścieżka bezwzględna i nazwa pliku w folderze z .exe-kiem zadziałają.